File: beginner/1115 - quadrant/Quadrant.go
package main

import (
	"fmt"
)

func main() {
	var x, y int
	var quadrant string

	for {
		fmt.Scanln(&x, &y)

		if x == 0 || y == 0 {
			break
		}

		if x > 0 {
			quadrant = "primeiro"

			if y < 0 {
				quadrant = "quarto"
			}
		} else {
			quadrant = "segundo"

			if y < 0 {
				quadrant = "terceiro"
			}
		}

		fmt.Println(quadrant)
	}
}

File: beginner/1115 - quadrant/Quadrant.java
import java.util.Scanner;
import java.io.IOException;

public class Quadrant {
	public static void main(String[] args) throws IOException {
		Scanner reader = new Scanner(System.in);
		int x, y;
		String quadrant = "";

		while (true) {
			x = reader.nextInt();
			y = reader.nextInt();

			if (x == 0 || y == 0) {
				break;
			}

			if (x > 0) {
				quadrant = y > 0 ? "primeiro" : "quarto";
			} else {
				quadrant = y > 0 ? "segundo" : "terceiro";
			}

			System.out.println(quadrant);
		}

		reader.close();
	}
}
